src/components/thumbnail-maker/TagItem.tsx

```
@@ -37,26 +38,31 @@ export function TagItem({ tag, onRemove, onClick, className }: Props) {
       style={tagStyle}
     >
       <span>
-        {tag.tagContentType === "3d-emoji" ? (
+        {tag.content.type === "3d-emoji" ? (
           <img
-            src={`https://avahrjwyynzeocqpyfhw.supabase.co/storage/v1/object/public/3d-fluent-emojis${tag.text.image}`}
+            src={get3DEmojiImage(tag.content.value)}
             width={90}
             height={90}
             style={{ pointerEvents: "none" }}
           />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.

(lint/a11y/useAltText)